solutions/gold/cses-1670.mdx

```diff
@@ -86,5 +96,99 @@ int main() {
 ```
 
 </CPPSection>
+</LanguageSection>
+</Spoiler>
+
+
+<Spoiler title="Solution 2">
+<Warning>
+While this solution is much easier to implement, it is extremely difficult to write an implementation that does not TLE in Java or Python. It may also even TLE in C++ if written inefficiently.
+</Warning>
```

Comment:
Why is this solution good then? If it doesn't easily pass in C++/Java, I'm not sure it's worth adding

Reply:
There are many CSES (and even USACO) problems that TLEs using Java/Python or an inefficient C++ code even with the right approach. (e.g. https://cses.fi/problemset/task/1635/)
![image](https://github.com/cpinitiative/usaco-guide/assets/52391257/87cb61a8-3e09-413b-9e42-adc3344dd524)
(this code runs the biggest test case in ~1300ms, and I had to do a small optimization by using constant variables)

In terms of educational value, it also encourages readers to practice coefficient optimization in my opinion. The approach I gave was to use unordered_set, but there more

Reply:
if this is an intended solution, then it's fine the way it is

if not, maybe replace title with "Slower Solution"

Reply:
```suggestion
<Warning>
While this solution is much easier to implement, it is extremely difficult to write an implementation that does not TLE in Java or Python. It may TLE in C++ if written inefficiently.
</Warning>
```